packages/roosterjs-editor-plugins/lib/plugins/ImageEdit/editInfoUtils/applyChange.ts

```diff
@@ -73,9 +74,28 @@ export default function applyChange(
     image.width = targetWidth;
     image.height = targetHeight;
```

Comment:
these 4 lines can be integrated into setImageResponsive()

Reply:
I cannot remove the width/height/style.width/style.height, because if they are remove will be not possible to use resize by the percentage. The image has to have  width/height/style.width/style.height, even if not resized.

Reply:
that is not true.  you can try insert a new image, at beginning it doesn't have these attributes/CSS at all, but you can still resize by percentage.Resizing by percentage is using image's naturalWidth/Height which is not an HTML attribute.

Reply:
`export default function resizeByPercentage(
    editor: IEditor,
    image: HTMLImageElement,
    percentage: number,
    minWidth: number,
    minHeight: number
) {
    const editInfo = getEditInfoFromImage(image);

    if (!isResizedTo(image, percentage)) {
        loadImage(image, editInfo.src, () => {
            if (!editor.isDisposed() && editor.contains(image)) {
                const lastSrc = image.src;
                const { width, height } = getTargetSizeByPercentage(editInfo, percentage);
                editInfo.widthPx = Math.max(width, minWidth);
                editInfo.heightPx = Math.max(height, minHeight);

                editor.addUndoSnapshot(() => {
                    applyChange(editor, image, editInfo, lastSrc);
                }, ChangeSource.ImageResize);
            }
        });
    }
}
`

resizeByPercentage uses applyChanges, so applyChanges needs to set the value o width/height/style;width/style.height, even if it is not resized by the resizer handles